### ISSUE TITLE
Checkpoint #3

### ISSUE THREAD

**Issue by redfoxanna** [redfoxanna/diamond-poems#7]
Hi @pawaitemadisoncollege! I have checkpoint 3 ready for you to take a look at. I still have a couple things I'm finishing up from the last checkpoint: refactoring for the generic dao and adding the genre table so I can have another 1:many for the project requirements. I hope to finish that up by the end of this week!

**Comment by pawaitemadisoncollege** [comment]
Hi @redfoxanna! 

When I signed up and logged in to your deployed app, I was redirected to localhost. My guess is that maybe something changed out there between the time you opened this issue and when I actually went out to try things out, but wanted to let you know just in case you weren't aware of it:
![Screenshot 2024-03-27 at 11 43 22 AM](https://github.com/redfoxanna/diamond-poems/assets/13874466/3f38a699-aa46-46ce-be11-50a57e91995e)

I poked around in your repo a little bit and am anxious to see/hear more about the S3/Textract work you are doing! Exciting!!!

**Comment by redfoxanna** [comment]
@pawaitemadisoncollege oh no! I see that you were registered as pawaite when I go to the console in AWS but now I'm seeing errors too. I did make some changes over the last few days and when I redeployed just now and tried to sign in I saw this: 

![Screenshot 2024-03-27 164942](https://github.com/redfoxanna/diamond-poems/assets/119988287/a4697367-25e8-49be-a052-b1140d692c16)

It is interesting because the timestamp for the token is the same time I'm making the request so it's almost like the token is expiring right away?
